src/components/Header/Header.scss

```diff
@@ -0,0 +1,45 @@
+.head {
+  align-items: center;
+  background: #f5f6f7;
+  border-bottom: solid 1px #dfe0e1;
+  display: flex;
+  line-height: 30px;
+  padding: 5px;
+
+  section {
+    display: flex;
+
+    &.nav {
+      border-right: solid 1px #dfe0e1;
```

Comment:
this should have `background: #F3F4F6`, no border necessary

also make this `height: 100%`